addons/pylons/XEH_postInit.sqf

```diff
@@ -2,28 +2,26 @@
 
 private _filter = "isClass (_x >> 'Components' >> 'TransportPylonsComponent') && {(getNumber (_x >> 'scope')) > 0}";
 GVAR(aircraftWithPylons) = (_filter configClasses (configFile >> "CfgVehicles")) apply {configName _x};
```

Comment:
These two lines need 160ms to run (with my modset) 
Might be worth moving that to preStart and caching it.